Makefile

```diff
@@ -1643,15 +1643,15 @@ backport:
 .PHONY: ensure-js-deps
 ensure-js-deps:
 	@if [[ "${WEBASSETS_SKIP_BUILD}" -eq 1 ]]; then mkdir -p webassets/teleport && touch webassets/teleport/index.html; \
-	else yarn install --ignore-scripts; fi
```

Comment:
`--ignore-scripts` was causing Connect on Linux not to have `pty-node` (I don't know why it worked with yarn).